config/env/development.js

```diff
@@ -2,7 +2,7 @@ var secret;
 try {
     secret = require('../../secret.config');
 } catch (err) {
-    secret = require('../../secret.config.sample');
+    secret = require('../../secret.config.json');
```

Comment:
I can see how this may have caused you confusion. Doing this makes the catch statement pointless, though. You will catch the error, then repeat the same code again, cause the same error, and not catch it.

A better solution would be to replace the code inside the catch statement with this:
`    secret = {
        secret: 'secret',
        emailUsername: '',
        emailPassword: ''
    };`